microbootstrap/bootstrappers/fastapi.py

```diff
@@ -29,11 +30,32 @@ class FastApiBootstrapper(
     application_config = FastApiConfig()
     application_type = fastapi.FastAPI
 
+    @contextlib.asynccontextmanager
+    async def _lifespan_manager(self, _: fastapi.FastAPI) -> typing.AsyncIterator[dict[str, typing.Any]]:
+        try:
+            self.console_writer.print_bootstrap_table()
+            yield {}
+        finally:
+            self.teardown()
+
+    @contextlib.asynccontextmanager
+    async def _wrapped_lifespan_manager(self, app: fastapi.FastAPI) -> typing.AsyncIterator[dict[str, typing.Any]]:
+        assert self.application_config.lifespan  # noqa: S101
+        async with self._lifespan_manager(app), self.application_config.lifespan(app):
+            yield {}
+
+    def _choose_lifespan_manager(
```

Comment:
I'd inline content of this function

Reply:
https://github.com/community-of-python/microbootstrap/pull/55

Reply:
Thanks!